swig/StringArray.hpp

```diff
@@ -0,0 +1,146 @@
+#ifndef __STRING_ARRAY_H__
```

Comment:
We don't check `swig` folder content for style issues, so I'll not leave any comments here, except the need of copyright info.
```suggestion
/*!
 * Copyright (c) 2020 Microsoft Corporation. All rights reserved.
 * Licensed under the MIT License. See LICENSE file in the project root for license information.
 */
#ifndef __STRING_ARRAY_H__
```